packages/run-it/src/components/ShowResponse/responseUtils.tsx

```diff
@@ -59,7 +59,7 @@ const ShowText = (response: IRawResponse) => (
  * @returns Image content
  */
 const imageContent = (response: IRawResponse) => {
-  let content
+  let content = ''
```

Comment:
Not sure that this change (the else also sets content) was needed but I'm okay with it

Reply:
the linter was complaining so I put that in to shut it up